Sources/Helpers/PrepareForSkeletonProtocol.swift

```diff
@@ -12,22 +12,50 @@ protocol PrepareForSkeleton {
     func prepareViewForSkeleton() 
 }
 
-extension UILabel: PrepareForSkeleton {
-    func prepareViewForSkeleton() {
-        text = " "
-        resignFirstResponder()
+extension UIView: PrepareForSkeleton {
+    @objc func prepareViewForSkeleton() {
+        _ = startTransitionIfAvailable()
+    }
+    fileprivate func startTransitionIfAvailable() -> Bool {
```

Comment:
since this method could be async, we could use a completion block instead of returns a Bool. Something like this:

```swift
typealias TransitionCompletionBlock = () -> Void

func startTransitionIfAvailable(_ completion: TransitionCompletionBlock? = nil)  {
    // do something
}

```